api/openapi/v2/system-agent.yaml

```diff
@@ -0,0 +1,625 @@
+      type: object
+      properties:
+        service:
+          description: "The service from which to obtain metrics"
+          type: string
+          example: "edgex-core-data"
+    GetConfigResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "Provides a response containing the configuration for the targeted service."
+      type: object
+      properties:
+        config:
+          description: "A string-ified representation of the service's configuration. For purposes of this specification, a string has been used since configuration structure differs from service to service."
+          type: string
+      required:
+        - config
+    HealthRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      properties:
+        services:
+          description: "The services from which to obtain health information."
+          type: array
+          items:
+            description: "The name of the service from which to obtain health information."
+            type: string
+            example: "edgex-core-data"
+    HealthResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      type: object
+      description: "Response containing the health status for the targeted service(s)."
+      additionalProperties:
+        type: string
+    MetricsRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      properties:
+        services:
+          description: "The services from which to obtain metrics."
+          type: array
+          items:
+            description: "The name of the service"
+            type: string
+            example: "edgex-core-data"
+    MetricsResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      properties:
+        sucess:
+          description: "The result of the metrics retreival process"
+          type: boolean
+          example: true
+        executor:
+          description: "The type of executor used to process the retrieve the metrics"
+          type: string
+          example: "docker"
+        operation:
+          description: "The type of operation performed"
+          type: string
+          example: "metrics"
+        service:
+          description: "The service assocaited with the metrics"
+          type: string
+          example: "edgex-core-data"
+        result:
+          description: "The metric information retireved from the targeted service"
+          type: object
+          additionalProperties:
+            type: object
+    OperationRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      description: "An Operation for SMA processing."
+      properties:
+        action:
+          description: "The operation to execute on the service."
+          type: string
+          enum: [start, stop, restart]
+        services:
+          description: "The services which to apply the action."
+          type: array
```

Comment:
Since it'd be possible in v2 to send multiple use-case requests in a single transport request, I wonder if we still need to support requesting for more than one service in a use-case request.